opennms-base-assembly/src/main/filtered/bin/update-package-permissions

```diff
@@ -27,12 +27,6 @@ if ! id -u "${RUNAS}" >/dev/null 2>&1; then
   exit 1
 fi
 
-if [ "${RUNAS}" != "opennms" ]; then
-  echo "\$RUNAS has been changed from the default 'opennms' user. Assuming you know what you're doing and _not_ changing ownership of any files."
-  echo ""
-  exit 0
-fi
```

Comment:
Maybe it would be better to leave the warning in, but not exit?

Reply:
I thought about doing that. I'll include it in the updated PR against foundation-2024 later this morning.